sql/telemetry_derived/main_summary_v4/part1.sql

```diff
@@ -272,9 +272,9 @@ SELECT
   environment.partner.distribution_id,
   DATE(submission_timestamp) AS submission_date,
   -- See bug 1550752
-  udf_boolean_histogram_to_boolean(payload.histograms.fxa_configured) AS fxa_configured,
+  ifnull(environment.services.account_enabled, udf_boolean_histogram_to_boolean(payload.histograms.fxa_configured)) AS fxa_configured,
```

Comment:
Apparently IFNULL is a synonym for COALESCE. I'm more familiar with COALESCE. Any argument for preferring one over the other?

Reply:
Should we update the bug numbers to the ones that describe the new values?

Reply:
i like using `IFNULL` when it's only 2 fields, because it's shorter. i'm okay with preferring to never use `IFNULL`.

Reply:
i'm going to merge this with `IFNULL` for the moment, and file a follow-up issue to consider banning `IFNULL` in favor of `COALESCE`.

Reply:
> i like using `IFNULL` when it's only 2 fields, because it's shorter. i'm okay with preferring to never use `IFNULL`.

If it's shorter only in that the name of the function is fewer characters, I'd prefer to use COALESCE everywhere as the more general function.